content/about/can-tor-help-against-viruses/contents.lr

```diff
@@ -0,0 +1,11 @@
+_model: question
+---
+title: Can Tor help you against viruses?
+---
+seo_slug: use-tor-against-viruses
```

Comment:
I don't like this seo_slug since that's not Tor's goal and might confuse new users.